Update libraries.mdx

conor-mccullough opened the pull request:
Hello! We now maintain a dotnet client for Vault, updating the docs to include this info.

🔍 [Deploy preview](https://vault-git-conor-dotnet-patch-1-hashicorp.vercel.app/vault/api-docs/libraries)

maxb commented:
I wonder if it might be a bit premature to add this now, or if a warning about the very early stage of development should be included, if adding?

The autogenerated client libraries based on the Vault OpenAPI still have lots of methods which do not work, because the autogeneration and OpenAPI spec are not sufficiently developed yet.

Of the two that exist, vault-client-go has been getting more recent releases as development proceeds, whilst vault-client-dotnet has not been being released much at all.

maxb commented on website/content/api-docs/libraries.mdx:
````diff
@@ -36,6 +36,16 @@ $ go get github.com/hashicorp/vault/api
+- [Vault DotNet Client](https://github.com/hashicorp/vault-client-dotnet)
+
+```shell-session
+# Install-Package HashiCorp.Vault -version "0.1.0-beta"
````
None of the other libraries on this page mention a specific version number, so this one probably should not either.

conor-mccullough commented:
Good points, cheers. I'll make some modifications in a little and resubmit.

conor-mccullough commented:
Updated this to include a note about it being in early development, removed the -version flag. Also modified the `ansible-vault` description formatting to match.
